src/_P037_MQTTImport.ino

```diff
@@ -73,8 +73,11 @@ void Plugin_037_update_connect_status() {
     }
     if (!connected) {
       // workaround see: https://github.com/esp8266/Arduino/issues/4497#issuecomment-373023864
-      espclient_037 = WiFiClient();
-      espclient_037.setTimeout(CONTROLLER_CLIENTTIMEOUT_DFLT);
+      if (MQTTclient_037 != NULL) {
+        espclient_037 = WiFiClient();
+        espclient_037.setTimeout(CONTROLLER_CLIENTTIMEOUT_DFLT);
+        MQTTclient_037->setClient(espclient_037);
+      }
```

Comment:
@TD-er and what if it is null?

Reply:
Not sure, but if that happens, it may already happen right now.
I just added the setClient call and thus had to make a check for the pointer.